docs/pages/guides/local-app-development.mdx

```diff
@@ -0,0 +1,248 @@
+---
+title: Local app development
+description: Learn how to compile and build your app locally when using Expo.
+---
+
+import { BoxLink } from '~/ui/components/BoxLink';
+import { BookOpen02Icon, BuildIcon } from '@expo/styleguide-icons';
+import { CODE } from '~/ui/components/Text';
+import { Collapsible } from '~/ui/components/Collapsible';
+import { Step } from '~/ui/components/Step';
+import { Tabs, Tab } from '~/ui/components/Tabs';
+import { Terminal } from '~/ui/components/Snippet';
+
+To build your project into an app locally using your machine, you have to manually generate native code before testing the debug build or creating a production build for it to submit to the app store. There are two ways you can build your app locally. This guide provides a brief introduction to both methods and references to other guides that are necessary to create this workflow.
+
+## Prerequisites
+
+To follow the steps described in this guide, see the instructions below on how to set your development environment.
+
+<Collapsible summary="Android">
+
+<Tabs>
+
+<Tab label='macOS'>
+
+To set up your development environment on macOS for Android, you've to install Node.js, Watchman, JDK, and Android Studio.
```

Comment:
```suggestion
To set up your development environment on macOS for Android, you will install Node.js, Watchman, JDK, and Android Studio.
```

you've is not correct grammar here -- "you have" can't be abbreviated to "you've"